env/logging/log.go

```diff
@@ -0,0 +1,40 @@
+package logging
```

Comment:
This file can go away in favor of the logging package already in CTF: https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/logging/log.go
No need to have two separate ones

Reply:
good catch!